mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -260,6 +261,9 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_33190 canTakeDamage ()Z
 	METHOD method_33579 readFromPacket (Lnet/minecraft/class_2610;)V
 		ARG 1 packet
+	METHOD method_35053 isLeaping ()Z
+	METHOD method_35054 setLeaping (Z)V
```

Comment:
Sure, but I hope you can document that when an entity is "leaping", it does not experience drag (on its speed) horizontally

Reply:
I mean, this api is very likely used by mods, so documenting this would definitely help a lot.

Reply:
Fair point! In which case calling it "leaping" may not be the correct name to use (I thought it odd that this isn't in the goat class).